agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3748,8 +3748,12 @@ bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx)
         beerocks_header->actionhdr()->radio_mac() = hostap_params.iface_mac;
         beerocks_header->actionhdr()->direction() = beerocks::BEEROCKS_DIRECTION_CONTROLLER;
     }
-    return message_com::send_cmdu(master_socket, cmdu_tx, backhaul_params.controller_bridge_mac,
-                                  backhaul_params.bridge_mac);
+
+    auto dst_addr =
+        cmdu_tx.getMessageType() == ieee1905_1::eMessageType::TOPOLOGY_NOTIFICATION_MESSAGE
```

Comment:
Shouldn't we send with multicast to more message types?

Reply:
not from son_slave.